regolith/templates/preslist.tex

```diff
@@ -0,0 +1,37 @@
+\documentclass[prl,tighten,amsmath,amssymb,floatfix]{revtex4-1}
+
+
+% Packages
+\usepackage{graphicx}
+\usepackage{times}
+\usepackage{booktabs,dcolumn}
+\usepackage{siunitx}
+
+\setlength{\parindent}{0pt}
+
+% Document
+\begin{document}
+\textbf{Presentations}
+
+\textbf{ {{-latex_safe(pi['name']).upper() }}}
+\vskip 14pt
+
+\begin{enumerate}
+{% for pres in presentations %}
+\item {{ latex_safe(pres['authors']) }}, {\bf {{ latex_safe(pres['title']) }} },
```

Comment:
Can you do a list comprehension in the builder, apply `latex_safe`, and join? Then the jinja code could just be `pres['authors']`.
Eg. 
```python
pres['authors'] = ' '.join([latex_safe(a) for a in pres['authors']])
```

You can modify the join if you want a comma between the names.

Reply:
Actually one could do this in less code. You could do
```python
pres['authors'] = ', '.join(pres['authors'])
```
This will turn the authors into a string which `latex_safe` will properly operate upon.

Reply:
if it was me I think I would still wrap this inside the function.  I think there will be a bunch of cases where things will be strings or lists.  @scopatz  do you have a preference?